Library/SharedFunctions.swift

```diff
@@ -687,19 +687,15 @@ public func estimatedShippingText(
 
   guard estimatedMin > 0, estimatedMax > 0 else { return nil }
 
-  let currentCountry = project.stats.currentCountry ?? Project.Country.us
-
   let formattedMin = Format.currency(
     estimatedMin,
-    country: currentCountry,
-    omitCurrencyCode: project.stats.omitUSCurrencyCode,
```

Comment:
I think we still need to know if we should omit the us currency code. Don't we want to include `USD` if someone ex has their preferred currency set to canadian dollars and the project's currency is in usd? And not include it if the project and the user's currency are both usd?

Reply:
The designs don't include the currency code as well as the currency symbol so I removed this option. I can double check with Alison, unless you've had a conversation about this already.

Reply:
Yeah it looks like we have inconsistencies in a few places when it comes to showing or not showing currency codes. Both in the app and in the designs even. I'll put `omitUSCurrencyCode` back in here since this is the behavior that has been, but also make a note to call this out to alison and ask for more direction.

Reply:
Sounds good!